cmd/notary/tuf.go

```diff
@@ -163,8 +164,9 @@ func tufList(cmd *cobra.Command, args []string) {
 		fatalf(err.Error())
 	}
 
-	// Retreive the remote list of signed targets
-	targetList, err := nRepo.ListTargets(data.CanonicalTargetsRole, "targets/releases")
+	// Retrieve the remote list of signed targets, prioritizing the passed-in list over targets
+	roles = append(roles, data.CanonicalTargetsRole)
+	targetList, err := nRepo.ListTargets(roles...)
```

Comment:
I think notary itself should be completely customizable for adding/removing/listing targets from any created role, and shouldn't hardcode "targets/releases".  And even so, the order we have in notary right now will prioritize the targets role over "targets/releases," which is not desirable.

docker calls `ListTargets` directly with prioritizing "targets/releases" (falling back to targets role if releases doesn't exist), and in docker it makes more sense to hardcode "targets/releases" https://github.com/docker/docker/blob/master/api/client/trust.go#L313